pruntime/enclave/src/contracts/balance.rs

```diff
@@ -50,6 +72,9 @@ pub enum Response {
         #[serde(with = "super::serde_balance")]
         total_issuance: chain::Balance
     },
+    TransferToChain {
+        transfer_queue: Vec<Vec<u8>>,
```

Comment:
When returning any binary data as a json, better to encode it to base64 because [u8] is too long in a json string